src/Pages/ResultsPage/ResultsPage.tsx

```diff
@@ -0,0 +1,35 @@
+import React ,{useEffect} from 'react'
+import "./ResultsPage.css";
+import {useAppSelector,useAppDispatch} from "./../../Redux/hooks";
+import {calculateResult} from "./../../Redux/Reducers/FunctioningSlice";
```

Comment:
Suggestion: Use an absolute path instead of a relative path. You can refer to [absolute path in react](https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i#:~:text=Absolute%20Path%20is%20when%20you,located%20in%20the%20project%20structure.) if you want to implement

Reply:
Yeah but it was showing error That's why not done it